mlflow/transformers.py

```diff
@@ -504,22 +514,22 @@ def save_model(
 @docstring_version_compatibility_warning(integration_name=FLAVOR_NAME)
 @format_docstring(LOG_MODEL_PARAM_DOCS.format(package_name=FLAVOR_NAME))
 def log_model(
-    transformers_model,
-    artifact_path: str,
-    processor=None,
-    task: Optional[str] = None,
-    model_card=None,
-    inference_config: Optional[Dict[str, Any]] = None,
-    code_paths: Optional[List[str]] = None,
-    registered_model_name: str = None,
-    signature: Optional[ModelSignature] = None,
-    input_example: Optional[ModelInputExample] = None,
-    await_registration_for=DEFAULT_AWAIT_MAX_SLEEP_SECONDS,
-    pip_requirements: Optional[Union[List[str], str]] = None,
-    extra_pip_requirements: Optional[Union[List[str], str]] = None,
-    conda_env=None,
-    metadata: Dict[str, Any] = None,
-    **kwargs,
+        transformers_model,
+        artifact_path: str,
+        processor=None,
+        task: Optional[str] = None,
+        model_card=None,
+        inference_config: Optional[Dict[str, Any]] = None,
+        code_paths: Optional[List[str]] = None,
+        registered_model_name: str = None,
+        signature: Optional[ModelSignature] = None,
+        input_example: Optional[ModelInputExample] = None,
+        await_registration_for=DEFAULT_AWAIT_MAX_SLEEP_SECONDS,
+        pip_requirements: Optional[Union[List[str], str]] = None,
+        extra_pip_requirements: Optional[Union[List[str], str]] = None,
+        conda_env=None,
+        metadata: Dict[str, Any] = None,
+        **kwargs,
```

Comment:
Why not add argument to `save_model` method for these huggingface options ?

Reply:
We can add it for the chunk size, but I think there should still be a default ideally